libs/components/layout/src/lib/modules/box/box.component.ts

```diff
@@ -38,4 +58,16 @@ export class SkyBoxComponent {
    */
   @Input()
   public ariaRole: string | undefined;
+
+  @ViewChild('titleRef', { read: ElementRef })
+  public set boxHeaderRef(value: ElementRef | undefined) {
+    if (value) {
```

Comment:
Is this meant to only set the descriptor on the content info provider when the consumer specifies a box header? As this is written, `titleRef` references a `span` that will always be present regardless of whether a header is present inside it. You would need to use `@ContentChild(SkyBoxHeaderComponent)` to detect when a `sky-box-header` is present.

Another option would be to provide a service from `SkyBoxComponent` that `SkyBoxHeaderComponent` and `SkyBoxControlsComponent` can inject. `SkyBoxHeaderComponent` can push a value to an Observable on that service which `SkyBoxControlsComponent` listens for and sets the descriptor only for its children. I'll leave it to you to decide which implementation to use.